### ISSUE TITLE
Pilocytic astrocytoma

### ISSUE THREAD

**Issue by DLewisSmith** [obophenotype/human-phenotype-ontology#6740]
**Preferred term label:**
Pilocytic astrocytoma

**Synonyms**

**Definition (free text, please give PubMed ID)**
The most common form of astrocytoma (WHO Grade I) in childhood. These typically have MAPK signalling pathway abnormalities.

PMID: 23817572

**Parent term (use hpo.jax.org/app)**
Astrocytoma HP:0009592

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-1735-8178

**Comment by pnrobinson** [comment]
added, thanks, with xref NCIT:C4047
